rubix-bookkeeper/src/main/java/com/qubole/rubix/bookkeeper/FileDownloadRequestChain.java

```diff
@@ -87,7 +87,7 @@ public long getTimeSpentOnDownload()
     return this.timeSpentOnDownload;
   }
 
-  public Integer call() throws IOException
+  public Long call() throws IOException
```

Comment:
nit: we can use primitive long.

Reply:
Can't, this is derived from `implements Callable<Long>` of `ReadRequestChain`